packages/editor/src/store/actions.js

```diff
@@ -404,13 +393,16 @@ export function* undo() {
 }
 
 /**
- * Returns an action object used in signalling that undo history record should
- * be created.
+ * Action that creates an undo history record.
  *
- * @return {Object} Action object.
+ * @deprecated Since Gutenberg 13.0.0
  */
 export function createUndoLevel() {
-	return { type: 'CREATE_UNDO_LEVEL' };
+	deprecated( "wp.data.dispatch( 'core/editor' ).createUndoLevel", {
+		since: '5.9',
+		alternative: 'Use the core entities store instead',
```

Comment:
Is GitHub missing a commit or something, I don't see the "version": "6.0"

Reply:
Sorry, turns out I didn't really push the commit to the server 🙂 Please look again now.